components/Home.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import {
+  StyleSheet,
+  Text,
+  View,
+  ViewStyle,
+  TextStyle,
+  StyleProp,
+  TouchableOpacity,
+  ImageStyle
+} from 'react-native';
+
+interface IProps {
+  title?: StyleProp<TextStyle>;
+  button?: StyleProp<ViewStyle>;
+  navigation?: any;
```

Comment:
why is this any? lol

Reply:
navigation is an object that has the property of navigate. Should have specified navigation as an object and passed navigate to it, then specified the type of navigate in the object. Fixing it in the next PR.